### ISSUE TITLE
Fix coverage errors after multiple builds

### ISSUE THREAD

**Issue by hgducharme** [hgducharme/meatball#25]
The build outputs a wall of:

```
profiling: /Users/hgd/repos/meatball/build/./src/magicbitboard.gcda: cannot merge previous GCDA file: mismatched number of counters (14)
profiling: /Users/hgd/repos/meatball/build/./src/magicbitboard.gcda: cannot merge previous GCDA file: corrupt arc tag (0x00000000)
profiling: /Users/hgd/repos/meatball/build/./src/magicbitboard.gcda: cannot merge previous GCDA file: corrupt arc tag (0x00000000)
profiling: /Users/hgd/repos/meatball/build/./src/magicbitboard.gcda: cannot merge previous GCDA file: corrupt arc tag (0xb06f3549)
profiling: /Users/hgd/repos/meatball/build/./src/magicbitboard.gcda: cannot merge previous GCDA file: corrupt arc tag (0x01000000)
profiling: /Users/hgd/repos/meatball/build/./src/magicbitboard.gcda: cannot merge previous GCDA file: corrupt arc tag (0x00000002)
```

**Comment by hgducharme** [comment]
To reproduce:

1. Build unit tests

```
make unit_tests
```

2. Run the executable

```
./bin/unit_tests
```

3. Change the source code (e.g. delete the last test case from `test_move.cpp`)
4. Rebuild the unit tests

```
make unit_tests
```

5. Run the executable again

```
./bin/unit_tests
```

6. You should see the coverage errors

**Comment by hgducharme** [comment]
Fixed in #54
